lib/transaction.ex

```diff
@@ -18,7 +18,7 @@ defmodule Braintree.Transaction do
           additional_processor_response: String.t(),
           amount: number,
           android_pay_details: String.t(),
-          apple_pay_details: String.t(),
+          apple_pay: map,
```

Comment:
Is there a reference to when this change was made? Also, does it change `android_pay_details` as well?

Reply:
Here's a [line in the official Braintree Python lib](https://github.com/braintree/braintree_python/blob/89219d691eafe763a09ad28f04400c4fcef4991e/braintree/transaction.py#L696) showing this field name. [And here's the commit where it was added 5 years ago](https://github.com/braintree/braintree_python/commit/ac787db904fa8978002c18783b02c47c36e6981a). I think braintree-elixir has had the wrong name the whole time and nobody has noticed. Thanks for being careful and double checking before merging 😄 .

I have integration tests in my system that make requests to the Braintree Sandbox, and I have test coverage verifying this PR as well as #54 and #57. I included a test in #57, but this PR, #54, and Google Pay could use coverage.

For Google Pay, it looks like the field is [`android_pay_card`](https://github.com/braintree/braintree_python/blob/89219d691eafe763a09ad28f04400c4fcef4991e/braintree/transaction.py#L700).

I'll try to find time today to do the following:

* Add integration test coverage to this PR (or separately if you merge it first).
* Create a pull-request with integration test coverage for #54.
* Create a pull-request for `android_pay_card` with integration test coverage.

Thanks for maintaining this library!